yarn.lock

```diff
@@ -19015,7 +19015,7 @@ __metadata:
     "@angular/platform-browser-dynamic": ^15.2.9
     "@angular/router": ^15.2.9
     "@edium/fsm": ^2.1.2
-    "@hmcts/ccd-case-ui-toolkit": 7.0.10
+    "@hmcts/ccd-case-ui-toolkit": 7.0.8-compress-ls
```

Comment:
This is probably not the version of toolkit you want. The live version is 7.0.10